config/gradle/common.gradle

```diff
@@ -43,3 +43,21 @@ tasks.javadoc {
     // TODO: Temporary until javadoc has been fixed for Java 8 everywhere
     failOnError = false
 }
+
+apply plugin: 'jacoco'
+
+jacoco {
+    toolVersion = "0.8.13"
+}
+
+test {
+    finalizedBy jacocoTestReport
+}
+
+jacocoTestReport {
+    dependsOn test
+    reports {
+        xml.required = true
+        html.required = true
+    }
+}
```

Comment:
Did we need to add this here? If my understanding is correct, this will force `jacoco` reports to be generated for all tests in modules that inherit their configuration from `common.gradle`. I thought that was an expensive operation before but things may have moved on.